iq/questions/tests.py

```diff
@@ -63,20 +50,40 @@ def test_valid_question_form(self):
             "difficulty": "5",
             "tags": "asdf,fdsa,asdfasdf"
         })
-        form.is_valid()
+        self.assertTrue(form.is_valid())
         count = Question.objects.count()
         form.save()
         self.assertEqual(count+1, Question.objects.count())
 
-    def test_valid_question_form_with_category(self):
-        count = Question.objects.count()
-        question = make(Question)
-        form = QuestionForm(category_id=self.category.pk, instance=question, data={
+    def test_valid_question_form_with_new_category(self):
+        q = make(Question)
+        c = make(Category)
+        form = QuestionForm(instance=q, data={
             "body": "foo",
+            "categories": [c.pk] # Assign new category
         })
         self.assertTrue(form.is_valid())
-        form.save()
-        self.assertEqual(count+1, Question.objects.count())
+        saved_question = form.save()
+        self.assertEqual(
```

Comment:
Wouldn't a more accurate/readable assert be something like `self.assertTrue(CategoryQuestion.objects.filter(category=c, question=saved_question).exists())`. You just want to ensure the Question was added to the Category, right?